Gemfile

```diff
@@ -21,7 +21,8 @@ gem "delayed_job_active_record", "~> 4.1.7"
 gem "devise", "~> 4.8.1"
 gem "devise-security", "~> 0.17.0"
 gem "file_validators", "~> 3.0.0"
-gem "font-awesome-sass", "~> 5.15.1" # Remember to update vendor/assets/images/fontawesome when updating this gem
+gem "font-awesome-sass", "~> 6.4.0"
+# Remember to update vendor/assets/images/fontawesome when updating this gem
```

Comment:
Sorry, I forgot one thing! :pray:

Could you remove the commit changing this line? You can ignore the warning in this case.

The thing is, it's much easier for developers like me to accidentally ignore the comment if it's on a different line. If it's on the same line, even I'm going to notice it when upgrading (or, at least, I hope I will! :crossed_fingers:).